### ISSUE TITLE
Missing build files

### ISSUE THREAD

**Issue by marillat** [qyot27/libutvideo#4]
14.2.1 and 15.0.0 tarballs doesn't contains build files (config.guess, config.sub, configure, GNUmakefile, version.sh and README) so these tarball are useless without your work :)

**Comment by tokiclover** [comment]
Yes, build files should be included in master branch and in every tag/tarball.
Thanks.

**Comment by qyot27** [comment]
No, the master branch is not for the buildsystem.  It's a straight mirror of the official Ut Video source zips from http://umezawa.dyndns.info/archive/utvideo and the branches are separate because this repo was only meant to keep the buildsystem alive in an expanded form (the buildsystem which was removed from the official tarballs quite some time ago).  I do practically nothing on the master branch, save for tagging, in order to keep the history clean.

Also, the official Ut Video source finally was put up on Github with full commit history, [here](https://github.com/umezawatakeshi/utvideo).  Efforts would likely be far better served reforking from it, adding a CMake-based build system so there's no duplication or hacked-apart wonky configure script (not to mention support for Ninja), and open pull requests to fix any remaining issues building as a library with GCC.

And considering the fact that 15.2.0 no longer even compiles using the buildsystem branch (although other fixer repos have apparently sprung up independently which may or may not resolve that, sans configure), it's increasingly a moot point.

Generally speaking, FFmpeg is the *only* thing I'm aware of that even uses libutvideo as a system library rather than as a media framework component, and aside from the Ut Video Pro formats that support >8bit which still have not been added to the native utvideo decoder and encoder in libavcodec, there's really no need to use libutvideo anymore (and even then, the only thing the libutvideo wrapper in libavcodec can do is decode the format, not encode).
